resources/para-200-genesis

```diff
@@ -0,0 +1 @@
+0x0000000000000000000000000000000000000000000000000000000000000000006eceeb89f3ba476aff716ddb165a3aa307dcb0c5fa6348d067990e446b55f7f403170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c11131400
```

Comment:
This file and wasm file should not in the git repo.

Reply:
while initializing a prarachain on polkadot.js we need this `genesis` data and the `wasm` file, I saw that some others repo are keeping them as well.